provision/vvv-init.sh

```diff
@@ -1,5 +1,19 @@
 # Provision WordPress Develop
 
+WP_TYPE=`get_config_value 'wp_type' "single"`
+
+if [ "${WP_TYPE}" != "single" ]; then
+  SRC_DOMAINS="src.wordpress-ms-develop.dev *.src.wordpress-ms-develop.dev ~^src\.wordpress-ms-develop\.\d+\.\d+\.\d+\.\d+\.xip\.io$"
+  BUILD_DOMAINS="build.wordpress-ms-develop.dev *.build.wordpress-ms-develop.dev ~^build\.wordpress-ms-develop\.\d+\.\d+\.\d+\.\d+\.xip\.io$"
+  SRC_DOMAIN="src.wordpress-ms-develop.dev"
+  BUILD_DOMAIN="build.wordpress-ms-develop.dev"
+else
+  SRC_DOMAINS="src.wordpress-develop.dev *.src.wordpress-develop.dev ~^src\.wordpress-develop\.\d+\.\d+\.\d+\.\d+\.xip\.io$"
+  BUILD_DOMAINS="build.wordpress-develop.dev *.build.wordpress-develop.dev ~^build\.wordpress-develop\.\d+\.\d+\.\d+\.\d+\.xip\.io$"
+  SRC_DOMAIN="src.wordpress-develop.dev"
+  BUILD_DOMAIN="build.wordpress-develop.dev"
+fi
+
 # Make a database, if we don't already have one
 echo -e "\nCreating database 'wordpress_develop' (if it's not already there)"
```

Comment:
You should probably change this to `wordpress_develop_TYPE` when setting up multisite so all configurations can be run side by side.